.gitignore

```diff
@@ -8,3 +8,7 @@ node_modules
 # Builds
 bin/
 roo-cline-*.vsix
+
+# Local prompts and rules
+prompts
```

Comment:
This leads to the system prompt getting ignored - can we remove it?

Reply:
are you sure?, doesn't look like it to me, this specified a `prompt` folder on the root of the repo
<img width="315" alt="image" src="https://github.com/user-attachments/assets/a250e410-4a27-4bc6-8101-09b3defc8a8c" />

Reply:
![Screenshot 2024-12-16 at 3 12 24 PM](https://github.com/user-attachments/assets/dc3e37f9-d670-4e6e-a58b-618d26758c37)

Reply:
From https://git-scm.com/docs/gitignore
>If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level.

Reply:
updated